test/specs/settings.e2e.ts

```diff
@@ -272,6 +272,8 @@ describe('@settings - Settings', () => {
       await sleep(1000);
       await tap('BackupWallet');
       await sleep(1000); // animation
+      await tap('BackupIntroViewContinue'); // proceed past backup intro screen
+      await sleep(1000);
```

Comment:
I think this is actually a bug I point out here: https://github.com/synonymdev/bitkit-android/pull/554#issuecomment-3696401303

Tests should remain unchanged.